cosmo/retrieval/ghosts.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env python
```

Comment:
I'm not sure that this is a good idea. I understand that this is meant to "clean up" the working codebase, but in moving the "unused" bits we may loose context which could make it harder to figure out what they were meant for. Maybe a better option would be to just comment out the unused code so that it retains the context in which it was written, but has no effect on the package.